components/Episode/EpisodeFunctions.js

```diff
@@ -0,0 +1,25 @@
+export function getCoverURL(links) {
+  if (links) {
+    const { cover_kosmos, cover_itunes, cover } = links;
+    if (cover_kosmos) {
+      return cover_kosmos;
+    } else if (cover_itunes) {
+      return cover_itunes;
+    } else if (cover) {
+      return cover;
+    } else {
+      return false;
+    }
```

Comment:
If the "cyclomatic complexity" (for example a lot of `else ifs`) gets high, it might be better to use a `switch case` statement.

Reply:
I tried to make it work using mdn, a youtube tutorial, and code sandbox but I couldn't figure it out.
So I'm going to leave the `if else` in there for now...